src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubBranchSourceListener.java

```diff
@@ -0,0 +1,56 @@
+package org.jenkinsci.plugins.github_branch_source;
+
+import com.google.inject.Inject;
+import hudson.Extension;
+import hudson.model.Item;
+import hudson.model.listeners.ItemListener;
+import java.util.logging.Logger;
+import jenkins.branch.OrganizationFolder;
+import jenkins.model.Jenkins;
+import jenkins.scm.api.SCMNavigator;
+
+/**
+ * Listens for updates to GitHub branch sources in order to auto-register web hooks
+ */
+@Extension
+public class GitHubBranchSourceListener extends ItemListener {
+    @Inject
+    private transient Logger logger;
```

Comment:
Why do you need to inject the logger? It could instantiated in the declaration itself.